### PR TITLE
LON-845

### PR DESCRIPTION
**Pull request by dfernandezramos**
Also fix creating a new Playlist by right clicking on an event.

### CODE REVIEW

File: VAS.UI.Gtk2/UI/Menu/PlaysMenu.cs [hunk 1]
```diff
@@ -25,6 +25,7 @@
 using VAS.Core.Events;
 using VAS.Core.Store;
 using VAS.Core.Store.Playlists;
+using VAS.Core.Interfaces;
```
Comment (vguzmanp) [inline VAS.UI.Gtk2/UI/Menu/PlaysMenu.cs]: Sort usings

File: VAS.UI.Gtk2/UI/Menu/PlaysMenu.cs [hunk 2]
```diff
@@ -115,9 +116,27 @@ static public void FillAddToPlaylistMenu (MenuItem addToPlaylistMenu, Project pr
 				foreach (Playlist pl in project.Playlists) {
 					item = new MenuItem (pl.Name);
 					plMenu.Append (item);
+					item.Activated += (sender, e) => {
+						IEnumerable<IPlaylistElement> elements = events.Select (p => new PlaylistPlayElement (p));
+						App.Current.EventsBroker.Publish (
+							new AddPlaylistElementEvent {
+								Playlist = pl,
+								PlaylistElements = elements.ToList ()
```
Comment (vguzmanp) [inline VAS.UI.Gtk2/UI/Menu/PlaysMenu.cs]: This could be a `IEnumerable` instead, to avoid using `ToList` everywhere

**Comment by dfernandezramos** [comment]
Ready to review again. @vguzmanp I finally left PlaylistElements as a list as we have talked

**Comment by vguzmanp** [comment]
:+1:

**Pull request merged**